src/dependency-manager/src/environment-config/builder.ts

```diff
@@ -54,6 +54,12 @@ export class EnvironmentConfigBuilder {
       throw new Error('Invalid file format. Must be json or yaml.');
     }
 
+    for (const key of Object.keys(js_obj)) {
```

Comment:
Should both of these go in buildFromJSON? Would hopefully prevent having to have client-side changes https://gitlab.com/architect-io/architect-hub/-/merge_requests/268/diffs.